backend/docs/frontend-google-calendar-integration.md

```
@@ -0,0 +1,343 @@
+  description: string | null;
+  source: 'NOTION' | 'GOOGLE';  // 신규 필드
+}
+```
+
+---
+
+## 4. UI 구현 가이드
+
+### 4.1 캘린더 연동 설정 페이지
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)